site/app/templates/submission/EditTeamName.twig

```diff
@@ -4,7 +4,10 @@
 {% block body %}
     <p class="std-margin">
         Please fill in an appropriate name for your team if you would like.
+        <br>
+        Team names are limited to 30 characters
```

Comment:
See comment below about passing the "30" from the backend so its always the same everywhere